content/docs/devs/contribution.md

```
@@ -1,403 +1,139 @@
 
-# Añadir cambios
-git add .
-
-# Hacer commit con mensaje descriptivo
-git commit -m "feat(audio): add volume normalization
-
-Implement automatic volume normalization to provide
-consistent output levels across different devices.
-
-Closes #1234"
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)